src/mysql.c

```diff
@@ -463,40 +463,40 @@ static int mysql_read_slave_stats(mysql_database_t *db, MYSQL *con) {
 
   field_num = mysql_num_fields(res);
   if (field_num < 33) {
-    ERROR("mysql plugin: Failed to get slave statistics: "
+    ERROR("mysql plugin: Failed to get replica statistics: "
           "`%s' returned less than 33 columns.",
           query);
     mysql_free_result(res);
     return -1;
   }
 
-  if (db->slave_stats) {
+  if (db->replica_stats) {
     unsigned long long counter;
     double gauge;
 
-    gauge_submit("bool", "slave-sql-running",
```

Comment:
Changing the metric names breaks backwards compatibility. Can we ear-mark removing this language from the metric identity for version 6 please?

Reply:
you are right, this is fixed in the next commit.